### DISCUSSION TITLE
Gradients and images support in test container

### DISCUSSION THREAD

**Discussion by tordex** [litehtml/litehtml#351] [Ideas]
When I made the flex support, I've found that many tests from [W3C](http://test.csswg.org/suites/css-flexbox-1_dev/nightly-unstable/) require gradients and images support. I've checked other test from W3C, gradients and images are used in other modules also.

So I guess we need to add support for images and gradients into the test container.

@stasoid, what do you think about this?

**Comment by stasoid** [comment]
I agree, gradients would be cool. I will look into this unless you want to.
Images are already supported, but only with 1-bit alpha. Do you mean to support 0-255 alpha or image formats other than png?

**Reply by tordex** [reply]
> I agree, gradients would be cool. I will look into this unless you want to.

It would be great if you look into this.

> Images are already supported, but only with 1-bit alpha. Do you mean to support 0-255 alpha or image formats other than png?

Oops, just notice images support. 1 bit alpha and png format is enough, I guess. Also the scaling image is required for tests because most tests scale images.

**Reply by stasoid** [reply]
> Also the scaling image is required for tests because most tests scale images.

This is also on me then.
